src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/reported/ReportsService.kt

```diff
@@ -44,14 +45,50 @@ class ReportsService(
     return reportedAdjudicationsPage.map { it.toDto() }
   }
 
-  fun getAdjudicationsForIssue(agencyId: String, locationId: Long? = null, startDate: LocalDate, endDate: LocalDate, pageable: Pageable): Page<ReportedAdjudicationDto> {
-    val statuses = listOf(ReportedAdjudicationStatus.SCHEDULED, ReportedAdjudicationStatus.UNSCHEDULED)
+  fun getAdjudicationsForIssue(agencyId: String, locationId: Long? = null, startDate: LocalDate, endDate: LocalDate, issueStatuses: List<IssuedStatus>? = null, pageable: Pageable): Page<ReportedAdjudicationDto> {
+
+    val issueStatusesFilter = issueStatuses ?: IssuedStatus.values().toList()
+
     if (locationId == null)
-      return getAllReportedAdjudications(agencyId = agencyId, startDate = startDate, endDate = endDate, statuses = statuses, pageable = pageable)
+      return getAdjudicationsForIssueAllLocations(
+        agencyId = agencyId, startDate = startDate, endDate = endDate, issueStatuses = issueStatusesFilter, pageable = pageable
+      )
+
+    return getAdjudicationsForIssueOneLocation(
+      agencyId = agencyId, locationId = locationId, startDate = startDate, endDate = endDate, issueStatuses = issueStatusesFilter, pageable = pageable
+    )
+  }
+
+  private fun getAdjudicationsForIssueAllLocations(agencyId: String, startDate: LocalDate, endDate: LocalDate, issueStatuses: List<IssuedStatus>, pageable: Pageable): Page<ReportedAdjudicationDto> {
+    if (issueStatuses.containsAll(IssuedStatus.values().toList()))
+      return getAllReportedAdjudications(agencyId = agencyId, startDate = startDate, endDate = endDate, statuses = ReportedAdjudicationStatus.issuableStatuses(), pageable = pageable)
+    if (issueStatuses.contains(IssuedStatus.ISSUED))
```

Comment:
There's quite a few parameters and a long method call in each of these, to help with readability we could put a line break between them and in `getAdjudicationsForIssueOneLocation`?

Reply:
sure.  was allowing ktlint to decide, but it seemed to like the no lines.  Will break it out a bit